jobs/JGDAS_ATMOS_CHGRES_FORENKF

```diff
@@ -78,10 +78,10 @@ if [ ${RUN_ENVIR} = "nco" -o ${ROTDIR_DUMP:-NO} = "YES" ]; then
     export COMIN_OBS=${COMIN_OBS:-$(compath.py ${envir}/obsproc/${obsproc_ver})/${RUN}.${PDY}/${cyc}/atmos}
     export COMIN_GES_OBS=${COMIN_GES_OBS:-$(compath.py ${envir}/obsproc/${obsproc_ver})/${GDUMP}.${gPDY}/${gcyc}/atmos}
 else
-    export COMOUT="${ROTDIR}/${CDUMP}.${PDY}/${cyc}/atmos"
+    export COMOUT="${ROTDIR}/${CDUMP}.${PDY}/${cyc}/${COMPONENT}"
```

Comment:
Same as above

Reply:
Same as above

Reply:
But, it is in an atmos j-job, and paths on the other side of the else use "atmos" (having recently been changed because we are now favoring not using `${COMPONENT}` if it never varies, which it usually doesn't).

Reply:
Same argument can also apply in CDUMP, rCDUMP etc. There are many places in gfs package contain this kind approach. Recommend to keep the same approach here by keep the use of COMPONENT variable.